source/indexes/single-field-index.txt

```diff
@@ -0,0 +1,77 @@
+.. _kotlin-sync-single-field-index:
+
+====================
+Single-Field Indexes
+====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:manual:`Single-field indexes </core/index-single/>` are indexes with a reference to a
+single field within a collection's documents. They improve single field query and sort
+performance, and support :manual:`TTL Indexes </core/index-ttl>` that automatically remove
+documents from a collection after a certain amount of time or at a specific clock time.
+
+.. note::
+
+   The ``_id_`` index is an example of a single-field index. This index is automatically
+   created on the ``_id`` field when a new collection is created.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the ``sample_mflix``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The documents in this collection are modeled by the following {+language+} data class:
+
+.. literalinclude:: /includes/indexes/indexes.kt
+   :start-after: start-movie-class
+   :end-before: end-movie-class
+   :language: kotlin
+   :copyable:
+
+Create Single-Field Index
+-------------------------
+
+The following example creates an index in ascending order on the ``title`` field:
```

Comment:
Q:Are there options to create an index in any other order other than ascending and descending?

S: Clarify/state explicitly that you can create an index in either ascending or descending order.

Reply:
Will add to page overview.